internal/provider/features.go

```diff
@@ -264,6 +264,21 @@ func schemaFeatures(supportLegacyTestSuite bool) *pluginsdk.Schema {
 				},
 			},
 		},
+
+		"managed_disk": {
+			Type:     pluginsdk.TypeList,
+			Optional: true,
+			MaxItems: 1,
+			Elem: &pluginsdk.Resource{
+				Schema: map[string]*pluginsdk.Schema{
+					"no_downtime_resize": {
```

Comment:
it appears the product name for this has settled on `Expand Without Downtime` as such we should make this:

```suggestion
					"expand_without_downtime": {
```